worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +53,31 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	std::string inputWord;
+	bool winGame = false;
+
+	std::cout << "The password is " << secret << "\n";
+
+	for (int numLives = 5; numLives > 0; numLives = numLives - 1) {
+		std::getline(std::cin, inputWord);
+		int output = getLikeness(inputWord, secret);
```

Comment:
The flowchart specifies a check that the entered word is in the list of candidates -- you have missed this